tests/common/devices/sonic.py

```diff
@@ -1101,8 +1101,8 @@ def get_uptime(self):
     def get_networking_uptime(self):
         start_time = self.get_service_props("networking", props=["ExecMainStartTimestamp", ])
```

Comment:
Looks like systemctl ExecMainStartTimestamp allways gives back in the system's local timezone and not UTC. 

So f we change the  self.get_now_time() to UTC, we need to convert start_time["ExecMainStartTimestamp"] to UTC as well